setup.py

```diff
@@ -60,70 +60,70 @@ def str_from_file(name):
     # transitive dependencies:
     #   urllib3: MIT
     #   aiohttp: Apache 2.0
-    "opensearch-py[async]==2.5.0",
+    "opensearch-py[async]>=2.5.0",
     # License: BSD
-    "psutil==5.8.0",
+    "psutil>=5.8.0",
     # License: MIT
-    "py-cpuinfo==7.0.0",
+    "py-cpuinfo>=7.0.0",
     # License: MIT
-    "tabulate==0.9.0",
+    "tabulate>=0.9.0",
     # License: MIT
-    "jsonschema==3.1.1",
+    "jsonschema>=3.1.1",
     # License: BSD
-    "Jinja2==3.1.3",
+    "Jinja2>=3.1.3",
     # License: BSD
-    "markupsafe==2.0.1",
+    "markupsafe>=2.0.1",
     # License: MIT
-    "thespian==3.10.1",
+    "thespian>=3.10.1",
     # recommended library for thespian to identify actors more easily with `ps`
     # "setproctitle==1.1.10",
     # always use the latest version, these are certificate files...
     # License: MPL 2.0
     "certifi",
     # License: Apache 2.0
-    "yappi==1.4.0",
+    "yappi>=1.4.0",
     # License: BSD
-    "ijson==2.6.1",
+    "ijson>=2.6.1",
     # License: Apache 2.0
     # transitive dependencies:
     #   google-crc32c: Apache 2.0
-    "google-resumable-media==1.1.0",
+    "google-resumable-media>=1.1.0",
     # License: Apache 2.0
-    "google-auth==1.22.1",
+    "google-auth>=1.22.1",
     # License: MIT
-    "wheel==0.38.4",
+    "wheel>=0.38.4",
     # License: Apache 2.0
     # transitive dependencies:
     #   botocore: Apache 2.0
     #   jmespath: MIT
     #   s3transfer: Apache 2.0
-    "boto3==1.28.62",
+    "boto3>=1.28.62",
     # Licence: BSD-3-Clause
-    "zstandard==0.22.0",
+    "zstandard>=0.22.0",
     # License: BSD
     # Required for knnvector workload
-    "h5py==3.10.0",
+    "h5py>=3.10.0",
     # License: BSD
     # Required for knnvector workload
-    "numpy==1.24.2",
+    "numpy>=1.24.2",
 ]
 
 tests_require = [
     "ujson",
-    "pytest==7.2.2",
-    "pytest-benchmark==3.2.2",
-    "pytest-asyncio==0.14.0"
+    "pytest>=7.2.2",
+    "pytest-benchmark>=3.2.2",
+    "pytest-asyncio>=0.14.0"
 ]
 
 # These packages are only required when developing Benchmark
 develop_require = [
-    "tox==3.14.0",
-    "coverage==5.5",
-    "twine==1.15.0",
-    "wheel==0.38.4",
-    "github3.py==1.3.0",
-    "pylint==2.6.0",
-    "pylint-quotes==0.2.1"
+    "tox>=3.14.0",
+    "coverage>=5.5",
+    "twine>=1.15.0",
+    "wheel>=0.38.4",
+    "github3.py>=1.3.0",
+    "pylint>=2.6.0",
```

Comment:
Try setting pylint back to `==` or restrict it to be under `3.X.X` as it's currently downloading version `3.1.0` and there's a CI error that seems to be related to Pylint `3.0+` versions
```
Collecting pylint>=2.6.0 (from opensearch-benchmark==1.5.0)
  Downloading pylint-3.1.0-py3-none-any.whl.metadata (12 kB)
```
Error
```
ImportError: cannot import name 'IAstroidChecker' from 'pylint.interfaces' (/home/runner/work/opensearch-benchmark/opensearch-benchmark/.venv/lib/python3.8/site-packages/pylint/interfaces.py)
```